doc/api/n-api.md

```diff
@@ -3216,6 +3216,35 @@ callback invocation, even if it has been successfully cancelled.
 
 ## Version Management
 
+### napi_get_node_version
+<!-- YAML
+added: REPLACEME
+-->
+
+```C
+typedef struct {
+  uint32_t major;
+  uint32_t minor;
+  uint32_t patch;
+  const char* release;
+} napi_node_version;
```

Comment:
would it make sense for this struct to also include the dependency version details equivalent to `process.versions`?